hazelcast/include/hazelcast/client/internal/nearcache/impl/store/BaseHeapNearCacheRecordStore.h

```diff
@@ -49,7 +49,6 @@ class BaseHeapNearCacheRecordStore
       ANCRS;
 
     BaseHeapNearCacheRecordStore(
-      const std::string& name,
```

Comment:
Could we revert this change? As it is public method, we should not change the signature.